luaui/Widgets/gui_buildmenu.lua

```diff
@@ -1391,54 +1383,54 @@ local function unbindBuildUnits()
 	boundUnits = {}
 end
 
-function widget:GameFrame(n)
+function widget:GameStart()
+	preGamestartPlayer = false
 
 	if checkGeothermalFeatures then
 		checkGeothermalFeatures()
 		checkGeothermalFeatures = nil
 	end
 
-	if preGamestartPlayer then
```

Comment:
Moving code out of `GameFrame()` and into `GameStart()`, we can't do this for all code there because the game needs to spawn the commander to transfer buildqueue